include/mbedtls/mbedtls_config.h

```diff
@@ -2534,16 +2534,40 @@
 #define MBEDTLS_ECJPAKE_C
 
 /**
- * \def MBEDTLS_ECP_C
+ * \def MBEDTLS_ECP_LIGHT
  *
  * Enable the elliptic curve over GF(p) library.
+ * This does not include support for scalar multiplication and functionalities
+ * that depepend on it, which include:
+ * - interruptible/restartable operations
+ * - reading of compressed EC points
+ * - parsing of explicit (non-named) curves
+ * - EC key derivation in PSA.
+ *
+ * \note This list of limitations is expected to change in the future as soon
```

Comment:
This should be a warning. (Usually we promise semantic stability for public APIs and config options, so exceptions need to be made clear.) Actually I'd put this warning right after the short description.
```
Enable part of the elliptic curve over GF(p) library.

\warning This option is experimental and its scope may change in future
versions. Most users will want to enable the stable option #MBEDTLS_ECP_C
instead. Enabling this option rather than #MBEDTLS_ECP_C is mostly useful to
minimize code size when you need none of #MBEDTLS_ECDH_C, #MBEDTLS_ECDSA_C, or
#MBEDTLS_ECJPAKE_C (for example because you have PSA drivers for them) but
still want support for (un)parsing ECC keys in the PK module.

\note Currently, this option enables the following parts of ecp.c: (un)parsing
and validation of keys, and info functions. It excludes scalar multiplication
and everything that depends on it, such as keypair generation.

```